### PR TITLE
Tan11389/readme changes through aug 4

### PR DESCRIPTION
**Pull request by tanneryould**
Please verify that I didn't accidentally add any invalid syntax. Also, I'm not sure if we want to include it in here, but I found a loose comparison and updated it to strict in one of the QML files.

### CODE REVIEW

File: ArcGISRuntimeSDKQt_CppSamples/EditData/EditFeaturesWithFeatureLinkedAnnotation/README.md [hunk 1]
```diff
@@ -35,9 +35,13 @@ Both expressions were defined by the data author in ArcGIS Pro using [the Arcade
 
 ## Offline data
 
-Link | Local Location
----------|-------|
-|[Loudoun geodatabase](https://arcgisruntime.maps.arcgis.com/home/item.html?id=74c0c9fa80f4498c9739cc42531e9948)|`<userhome>/ArcGIS/Runtime/Data/geodatabase/loudoun_anno.geodatabase`|
+1. Download the data from [ArcGIS Online](https://arcgisruntime.maps.arcgis.com/home/item.html?id=74c0c9fa80f4498c9739cc42531e9948).
```
Comment (ldanzinger) [inline ArcGISRuntimeSDKQt_CppSamples/EditData/EditFeaturesWithFeatureLinkedAnnotation/README.md:38]: I believe the offline data section is formatted a bit differently per SDK. In Qt, we use the table format. Also, the wording here with "scoped storage" is very Android specific - we try and keep it more platform agnostic in the Qt docs. I'd just revert this change (and the corresponding QML sample update) and stick with what we had

Comment (tanneryould) [reply ArcGISRuntimeSDKQt_CppSamples/EditData/EditFeaturesWithFeatureLinkedAnnotation/README.md]: Ok, good to learn. I've gone ahead and reverted the changes.

Comment (jared-2016) [reply ArcGISRuntimeSDKQt_CppSamples/EditData/EditFeaturesWithFeatureLinkedAnnotation/README.md]: I second what Lucas said here. Only thing you might consider adding is that on occasion we reference the samples readme for offline data, https://github.com/Esri/arcgis-runtime-samples-qt#use-offline-data-in-the-samples.

Here is an example, https://github.com/Esri/arcgis-runtime-samples-qt/tree/master/ArcGISRuntimeSDKQt_CppSamples/Scenes/Animate3DSymbols#offline-data.

**Review by ldanzinger — APPROVED** [review]
LGTM

**Review by jared-2016 — APPROVED** [review]
Just one opinion but either way, looks good!
